pkg/sloop/server/internal/config/testconfig.json

```diff
@@ -0,0 +1,44 @@
+{
+	"ConfigFile": "",
+	"leftBarLinks": null,
+	"resourceLinks": null,
+	"disableKubeWatch": false,
+	"kubeWatchResyncInterval": 1800000000000,
+	"webfilesPath": "./pkg/sloop/webfiles",
```

Comment:
please add a folder for mock data and these files there

Reply:
+1 to comment above, @kritikapradhan it is a good practice to move all test/mock data to a separate folder, so you won't accidentally use it in production.